gradle.properties

```diff
@@ -8,21 +8,28 @@
 #     http://www.apache.org/licenses/LICENSE-2.0
 #
 ### dependencies
+# For Nautilus 0-13-w11
```

Comment:
Please keep this list in alphabetical order.

Reply:
done